### DISCUSSION TITLE
Error: setting Connection Strings for Linux App Service

### DISCUSSION THREAD

**Discussion by jaymasc** [microsoft/PubSec-Info-Assistant#750] [Evening (PST) Office Hours (Q&A)]
I am deploying the information assistant for the first time. The initial build process is running and I am getting a new resource group and multiple resources within being deployed. About 25 minutes into the deployment I get this error:

│ Error: setting Connection Strings for Linux App Service (Subscription: "xx"
│ Resource Group Name: "infoasst-xx"
│ Site Name: "infoasst-func-tycnj"): unexpected status 400 with response: {"Code":"BadRequest","Message":"The parameter properties has an invalid value.","Target":null,"Details":[{"Message":"The parameter properties has an invalid value."},{"Code":"BadRequest"},{"ErrorEntity":{"ExtendedCode":"51008","MessageTemplate":"The parameter {0} has an invalid value.","Parameters":["properties"],"Code":"BadRequest","Message":"The parameter properties has an invalid value."}}],"Innererror":null}
│ 
│   ****with module.functions.azurerm_linux_function_app.function_app,
│   on core/host/functions/functions.tf line 82, in resource "azurerm_linux_function_app" "function_app":
│   82: resource "azurerm_linux_function_app" "function_app" {**
│** 
│ setting Connection Strings for Linux App Service (Subscription: "xx"
│ Resource Group Name: "infoasst-uscg003"
│ Site Name: "infoasst-func-tycnj"): unexpected status 400 with response: {"Code":"BadRequest","Message":"The parameter properties has an invalid value.","Target":null,"Details":[{"Message":"The
│ parameter properties has an invalid value."},{"Code":"BadRequest"},{"ErrorEntity":{"ExtendedCode":"51008","MessageTemplate":"The parameter {0} has an invalid
│ value.","Parameters":["properties"],"Code":"BadRequest","Message":"The parameter properties has an invalid value."}}],"Innererror":null}

I have made no modifications to the repository other than setting up the environment variables. Can anyone provide any insight into what is going wrong here?

**Comment by tauheed-zensar** [comment]
I am also getting the exact same issue. Please help me out.

**Reply by tauheed-zensar** [reply]
Also, if you see the environment variable under Azure function app service. Some variables from KeyVault marked as invalid. That's one of the observation i saw. I guess this is related to some access permission that we dont have but I am not completely sure.

**Reply by jaymasc** [reply]
It was my impression it had something to do with the variables for KeyVault as well. I have not figured it out yet unfortunately.

**Reply by tauheed-zensar** [reply]
yeah. let me know if you resolve it.

**Reply by jaymasc** [reply]
Did you ever figure this issue out?
